README.md

```diff
@@ -9,7 +12,6 @@
 * Open JDK
 * Python 3
 * Maven
-* Gradle
```

Comment:
Removed it cause it caused sometimes to download for too long when their servers were slow and with gradle wrapper this should not be needed.

Reply:
will leave it for now